packages/ui/src/components/Toasts/ToastContent.tsx

```diff
@@ -26,11 +23,11 @@ const ToastContent = ({ className, toast }: ToastContentProps) => {
     <div className={className}>
       <div className="iconContainer">
         {type === 'success' ? (
-          <CheckCircleOutlineIcon className="toastIcon" />
+          <HiOutlineCheckCircle size={20} />
         ) : type === 'loading' ? (
-          <CircularProgress className="toastIcon" />
+          <CircularProgress size={20} />
         ) : (
-          <ErrorOutlineIcon className="toastIcon errorIcon" />
+          <ErrorOutlineIconStyled size={20} />
```

Comment:
let's do size 25 here so that they are aligned with the text. 20 is a bit small.
![image](https://github.com/ChainSafe/Multix/assets/33178835/04f4ca39-ba23-4c38-8aef-95c9b2eff43b)